tvm/cli.py

```diff
@@ -57,22 +61,31 @@ def validate_version(ctx, param, value):  # pylint: disable=unused-argument
     return value
 
 
+def validate_version(ctx, param, value):  # pylint: disable=unused-argument
+    """Raise BadParameter if the value is not a tutor version."""
+    return version_is_valid(value)
```

Comment:
Is this function really necessary? Couldn't you use `version_is_valid` directly? I think `version_is_valid` and `validate_version` do the same. Please correct me if I'm wrong.

The rest looks good to me :D